README.md

```diff
@@ -6,5 +6,8 @@ This is the Proof of Humanity web app you will find at [https://app.proofofhuman
 
 Please see [CONTRIBUTING.md](./CONTRIBUTING.md).
 
-## Priority
-You can see which issues are a priority and help prioritize those using [tokenlog](https://tokenlog.xyz/proof-of-humanity/proof-of-humanity-web).
+### Issue Prioritization
+
+You can see a backlog of prioritized issues in our [Tokenlog](https://tokenlog.xyz/proof-of-humanity/proof-of-humanity-web).
+
+If you hold `$UBI` tokens, you can use your tokens for voting on which issues to implement next.
```

Comment:
@alcercu If someone is _not_ a registered `$UBI` recipient, but bought `$UBI` through an exchange, would they still be able to vote with Tokenlog?

Reply:
They wouldn't be able to. This is because this Tokenlog is based on a token called VOTE which is given to registered addresses on PoH, not `$UBI`. Also note that `$UBI` gives you voting power on UBI DAO, not PoH DAO.

Reply:
@alcercu I see, thanks. Where can I find the code that generates a VOTE token for registered addresses?

Reply:
@indie-0x676f  Here is the address of the contract https://etherscan.io/address/0x1c8eed9cd4fde67fc0f5171a9c684fbc7501170d but I don't know where is the code 🤔 I will try to get back to you with more info 👍🏻